tests/unit/test_python_type_checking.py

```
@@ -361,6 +361,114 @@ def test_strict_forces_all_rules_without_project_config(self, tmp_path):
+        assert "**/__pycache__" in exclude
+        # env/. env removed — too ambiguous (legitimate package names)
+        assert "**/env" not in exclude
+        assert "**/.env" not in exclude
+
+    def test_detected_venv_name_appended_to_excludes(self, tmp_path):
+        """Non-standard venv name is appended to excludes even if not in defaults."""
+        # Create a venv with a non-standard name — _detect_venv_path won't find
+        # it (only checks 'venv' and '.venv') so it won't be in BROAD_SCAN_EXCLUDES.
+        # But when it IS detected (via VIRTUAL_ENV env var) it should be appended.
+        import os
+        import unittest.mock as mock
```

Ruff (0.15.15)

[warning] Use `from unittest import mock` in lieu of alias

Replace with `from unittest import mock`

(PLR0402)